railties/lib/rails/generators/test_unit/authentication/templates/test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
+  test "new" do
+    get new_session_url
+    assert_response :success
+  end
+
+  test "create with valid credentials" do
+    post session_url, params: { email_address: "one@example.com", password: "password" }
```

Comment:
I left `"one@example.com"` hardcoded since `"password"` needs to be hardcoded here anyways, but I am happy to alternatively add a `setup { @user = User.take }` and do the following:

```ruby
    post session_url, params: { email_address: @user.email_address, password: "password" }
```

Reply:
Yes, think we should do the same `setup { @user = User.take }` as we do in the PasswordsControllerTest 👍

Reply:
(And then do as you suggest here, so the tests don't break when you start adding/changing the fixtures).